source/MaterialXCore/Document.h

```diff
@@ -352,6 +352,34 @@ class Document : public GraphElement
     /// Return a vector of all NodeDef elements that match the given node name.
     vector<NodeDefPtr> getMatchingNodeDefs(const string& nodeName) const;
 
+    /// @}
+    /// @name Backdrop Elements
+    /// @{
+
+    /// Add a Backdrop to the document.
+    BackdropPtr addBackdrop(const string& name = EMPTY_STRING)
```

Comment:
Can you not add Backdrops inside a NodeGraph? Is this a document only concept? Cause I would think you'd want them in NodeGraphs too.

Reply:
No, it can't be in a graph as it's not a node anymore. 
(Yes, It gives UI grouping and documentation only.)
The "contains" parameter instead gives a list of what it groups (+ documents) so it can be associated with any node or graph anywhere, including a node in a graph.